src/test/java/com/application/bookingservice/service/payment/PaymentServiceImplTest.java

```diff
@@ -0,0 +1,90 @@
+package com.application.bookingservice.service.payment;
+
+import static com.application.bookingservice.model.Payment.Status;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.application.bookingservice.dto.payment.PaymentResponseDto;
+import com.application.bookingservice.mapper.PaymentMapper;
+import com.application.bookingservice.model.Booking;
+import com.application.bookingservice.model.Payment;
+import com.application.bookingservice.repository.payment.PaymentRepository;
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class PaymentServiceImplTest {
+    @Mock
+    private static PaymentRepository paymentRepository;
+    @Mock
+    private static PaymentMapper paymentMapper;
+    @InjectMocks
+    private static PaymentServiceImpl paymentService;
+    private static Payment payment1;
+    private static Payment payment2;
+    private static PaymentResponseDto dto1;
+    private static PaymentResponseDto dto2;
+
+    @BeforeAll
+    static void beforeAll() {
+        payment1 = new Payment();
+        payment1.setId(1L);
+        payment1.setStatus(Status.SUCCEED);
+        Booking booking1 = new Booking();

```

Comment:
```suggestion
        Booking booking1 = new Booking();
```
create it also on the class level